Rocket.toml

```diff
@@ -1,3 +1,3 @@
-[development]
+[debug]
```

Comment:
From the [changelog](https://github.com/SergioBenitez/Rocket/blob/master/CHANGELOG.md#silent-changes):

> The default profile for debug builds is now `debug`, not `dev`.